src/core/utils/sort.ts

```diff
@@ -0,0 +1,8 @@
+import { orderAscFn } from "./orderAscFn";
+
+/**
+ * @internal
+ */
+export const sort = <T extends number>(arr: readonly T[]): T[] => {
+  return arr.slice().sort(orderAscFn);
```

Comment:
`slice` is slightly more efficient than `...` because of a simpler engine implementation IIRC